src/translator/byte_array_util.cpp

```diff
@@ -7,26 +7,26 @@ namespace marian {
 namespace bergamot {
 
 MemoryGift loadFileToMemory(const std::string& path, bool isModelFile){
-    std::ifstream is (path, std::ifstream::binary);
-    uint64_t length = 0; // Determine the length of file in bytes
-    if (is) {
-        is.seekg(0, is.end);
-        length = is.tellg();
-        is.seekg(0, is.beg);
-    } else {
-        std::cerr << "Failed opening file stream: " << path << std::endl;
-        std::exit(1);
+  std::ifstream is (path, std::ifstream::binary);
+  uint64_t length = 0; // Determine the length of file in bytes
+  if (is) {
+    is.seekg(0, is.end);
+    length = is.tellg();
+    is.seekg(0, is.beg);
+  } else {
+    std::cerr << "Failed opening file stream: " << path << std::endl;
+    std::exit(1);
```

Comment:
No that we're in `marian::bergamot`, `ABORT` macro? You can probably use `ABORT_IF` and a single line well.

Reply:
I thought ABORT was not there. Will change soon.